modelling/src/main/java/org/axonframework/modelling/command/AbstractRepository.java

```diff
@@ -76,6 +77,12 @@ protected AbstractRepository(Builder<T> builder) {
 
     @Override
     public A newInstance(Callable<T> factoryMethod) throws Exception {
+        return newInstance(factoryMethod, null);
```

Comment:
wouldn't it be better to use a no-op initMethod, instead of using null?

Reply:
We could do that and change the contract to not allow `null` and make the `initMethod` a hard requirement. I wanted to allow null to make it easier to use the API though, so I kept it like this. If you think making this argument `@NonNull` is better, I can change it.

Reply:
Wouldn't hurt to make it `@Nonnull`. 
However, I believe you've introduced that under 4.6.0, while we're dealing with 4.5.x here.

Apart from the use of the annotation, we have indeed steered away from `null` lambdas throughout the implementation, in favor of no-op methods.
From that stance, I'd wager it's better to go for a no-op indeed.